src/components/DataView/DataView.tsx

```diff
@@ -27,7 +27,7 @@ const StyledWrapper = styled.div`
 	& > .headerActions {
 		display: flex;
 		flex-direction: column;
-		padding: 20px;
+		padding: 8px 24px;

```

Comment:
I talked with Owen about the line below the grid actions. He said we should be changing it to whatever we feel looks best. So I think we should update .headerActions .grid padding-bottom: 8px;